.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- Great job installing linters correctly.

- [Optional] In all future projects, kindly ensure each commit message is written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).

- [Optional] Please in all future projects, kindly make sure you name your branch meaningfully to follow Github flow properly. Your branch name should be descriptive (e.g.: refactor-authentication, user-content-cache-key, hello-microverse-project) so that others can see what is being worked on.